src/extensions/replay/sessionrecording.ts

```diff
@@ -1290,7 +1288,6 @@ export class SessionRecording {
         }
 
         this._urlBlocked = false
-        document?.body?.classList?.remove('ph-no-capture')
```

Comment:
All the other changes make sense, this one does not feel related to the original issue?

Reply:
no, it's totally fly-by, commented on above. i don't think we should be doing that